library/classes/X12837P.php

```diff
@@ -0,0 +1,1239 @@
+<?php
```

Comment:
move this file to library/billing/src/ directory
(it will get automatically picked up by composer's autoload)